chainer/distribution.py

```diff
@@ -0,0 +1,420 @@
+import copy
+
+
+class Distribution(object):
+
+    """Interface of Distribution
+
+    `Distribution` is a bass class to treat probability distributions.
```

Comment:
Consider rewording: examples:

`Distribution` is a bass class for dealing with probability distributions.
`Distribution` is a bass class for handling probability distributions.
`Distribution` is a bass class for a probability distribution.